src/main/java/io/github/moulberry/notenoughupdates/options/separatesections/ItemOverlays.java

```diff
@@ -489,6 +489,31 @@ public class ItemOverlays {
 	@ConfigEditorInfoText()
 	public boolean customWitherCloakCredit = false;
 
+	@ConfigOption(
+		name = "Grappling hook Overlay",
+		desc = ""
+	)
+	@ConfigEditorAccordion(id = 9)
+	public boolean grappleAccordion = false;
+
+	@Expose
+	@ConfigOption(
+		name = "Enable grappling hook overlay",
+		desc = "Show the cooldown of grappling hook in the item durability"
+	)
+	@ConfigEditorBoolean
+	@ConfigAccordionId(id = 9)
+	public boolean enableGrappleOverlay = true;
+
+	@Expose
+	@ConfigOption(
+		name = "Enable armor check",
```

Comment:
im not sure why this needs to be a setting? (also you spelt armour wrong)

Reply:
Just made it because people want more customisability, it's enabled by default so no one should have issues with it

Reply:
the customisability option says "make this feature be wrong" doesn't make too much sense to me?

Reply:
ok imma remove after waking up from night shift